### DISCUSSION TITLE
Docs for multi-gpu training

### DISCUSSION THREAD

**Discussion by mlfffinder** [ACEsuit/mace#152] [Q&A] [answered]
Where can I  find the docs for multi-gpu training?
I heard from last question that it's in the README file, but cannot find it.  Which tag is for multi-gpu training?
+
Additional question for mace-layer. I don't really get what arbitrary 3D point clouds mean.
("Connstructing higher order equivariant graph neural networks for arbitrary 3D point clouds" in the readme)
Does it mean it is suitable to use mace-layer when i'm dealing the systems other than molecules , such as crystals ?

**Comment by ilyes319** [comment, marked as answer]
Sorry for the late reply.

The doc is in the readme of the multi-gpu branch.

Arbitrary means non atomistic one. The MACE package loads atomistic point cloud. If you want to deal with something else you can use mace-layer to your custum system (like 3D vision point cloud).
